pkg/option/config.go

```diff
@@ -3279,6 +3283,13 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 			ToFQDNsMaxDeferredConnectionDeletes)
 	}
 	switch {
+	case vp.IsSet(ToFQDNsActiveConnectionsTTL): // set by user
+		c.ToFQDNsActiveConnectionsTTL = vp.GetInt(ToFQDNsActiveConnectionsTTL)
```

Comment:
Again, here we are not checking that the user given value is actually >= `2*DNSGCJobInterval`, but I think we should.